src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateCopy = { ...state };
+  const arrayState = [];
```

Comment:
dont use data structure names when you create a variable
```suggestion
  const transformedStates = [];
```